datastore_entity/src/lib.rs

```diff
@@ -56,6 +56,10 @@ impl DatastoreProperties {
         Some(DatastoreProperties(entity.0.properties?))
     }
 
+    pub fn fromMap(map: BTreeMap::<String, Value>) -> DatastoreProperties {
```

Comment:
Oh no this is rust my friend, use `from_map` =)

Reply:
oh, yes yes